src/test/java/controllers/ApiControllerDocTesterTest.java

```diff
@@ -29,7 +29,7 @@
 public class ApiControllerDocTesterTest extends NinjaDocTester {
     
     String URL_INDEX = "/";
-    String URL_ACES_UP = "/AcesUp";
+    String URL_ACES_UP = "/AcesUp/English";
```

Comment:
Need another variable to test the spanish route and another to test for the welcome page maybe? Most likely just for the spanish version.